dvc/command/add.py

```diff
@@ -38,6 +43,15 @@ def add_parser(subparsers, parent_parser):
         default=False,
         help="Don't put files/directories into cache.",
     )
+    add_parser.add_argument(
+        "-f",
+        "--file",
+        help="Specify name of the stage file. It should be "
```

Comment:
Let's not copypaste it from `dvc/command/run.py`, but rather define it once and use in both places(maybe as a constant).

Reply:
:thinking: , it is nice to keep duplication at the minimum, but in this case is not exactly the same functionality (`dvc run --file` takes the **first** output, whereas `add` default's is a stage file per output).

I added a small message noting that it can't be used with multiple targets :)

Reply:
Right, didn't notice that right away. Thanks.